### PR TITLE
[Aftershock] misc typographical fixes

### PR DESCRIPTION
**Pull request by casswedson**
#### Summary
None

#### Purpose of change
To fix some small typographical errors, note that I am not a native English speaker, (I just happen to have a file with all the aftershock descriptions, vim's handy spell checker and the internet for references) so apologies for possible obvious mistakes.

Also the `afs_sentinel_cloak` description is missing a bit of information, since I am bad at descriptions myself I'll leave it to the aftershock maintainers, it's the first description changed so please leave a review fixing it.

#### Additional context
This one I am not sure about
https://github.com/CleverRaven/Cataclysm-DDA/blob/d87baa56d2d63cda1ae1146032081364b44212d5/data/mods/Aftershock/maps/furniture.json#L448
cheescutter?

### CODE REVIEW

**Comment by Maleclypse** [comment]
Google wire cheese slicer.

**Review by Maleclypse — APPROVED** [review]
Thanks!

**Comment by casswedson** [comment]
Thanks for making a great mod!

File: data/mods/Aftershock/items/items.json [hunk 1]
```diff
@@ -175,7 +175,7 @@
     "type": "AMMO",
     "category": "spare_parts",
     "name": { "str_sp": "titanium" },
-    "description": "A strong, durable metal.  Highly valued for its weight to strength ratio, it was an excellent upgrade to aluminum for those properties albeit a lot more expensive due to rarity.  More commonly used in jewelery and medical implants.",
+    "description": "A strong, durable metal.  Highly valued for its weight to strength ratio, it was an excellent upgrade to aluminum for those properties albeit a lot more expensive due to rarity.  More commonly used in jewellery and medical implants.",
```
Comment (actual-nh) [inline data/mods/Aftershock/items/items.json:178]:
```suggestion
    "description": "A strong, durable metal.  Highly valued for its weight to strength ratio and an excellent upgrade to aluminum for those properties - albeit a lot more expensive due to rarity.  More commonly used in jewelry and medical implants.",
```

File: data/mods/Aftershock/maps/furniture.json [hunk 2]
```diff
@@ -293,7 +293,7 @@
     "id": "f_crispr",
     "name": "CRISPR Biolab",
     "symbol": "&",
-    "description": "A boxy looking device of arms, droppers and pipettes.  It was used pre-Cataclysm to do expirements in gene editing.",
+    "description": "A boxy looking device of arms, droppers and pipettes.  It was used pre-Cataclysm to do experiments in gene editing.",
```
Comment (actual-nh) [inline data/mods/Aftershock/maps/furniture.json:296]:
```suggestion
    "description": "A boxy looking device of arms, droppers, and pipettes.  It was used pre-Cataclysm to do gene-editing experiments.",
```

File: data/mods/Aftershock/mobs/robots.json [hunk 3]
```diff
@@ -229,7 +229,7 @@
     "copy-from": "base_drone",
     "type": "MONSTER",
     "name": { "str": "bloodhound drone" },
-    "description": "A small spiked quadcopter robot equipped with a high-power spotlight.  This automated seeker drone was originally designed harry and harrass fugitives while maintaining visual contact for other pursuers.",
+    "description": "A small spiked quadcopter robot equipped with a high-power spotlight.  This automated seeker drone was originally designed harry and harass fugitives while maintaining visual contact for other pursuers.",
```
Comment (actual-nh) [inline data/mods/Aftershock/mobs/robots.json:232]:
```suggestion
    "description": "A small, spiked quadcopter robot equipped with a high-power spotlight.  This automated seeker drone was originally designed to harry and harass fugitives while maintaining visual contact for other pursuers.",
```

File: data/mods/Aftershock/mutations/mutations.json [hunk 4]
```diff
@@ -629,7 +629,7 @@
     "name": "Trumpeting Voice",
     "points": -1,
     "mixed_effect": true,
-    "description": "You have a trumpeting elephantine voice.  threatening NPCs will be easier, but lying will very hard.",
+    "description": "You have a trumpeting elephantine voice.  Threatening NPCs will be easier, but lying will very hard.",
```
Comment (actual-nh) [inline data/mods/Aftershock/mutations/mutations.json:632]:
```suggestion
    "description": "You have a trumpeting, elephantine voice.  Threatening NPCs will be easier, but lying will very hard.",
```

File: data/mods/Aftershock/player/professions.json [hunk 5]
```diff
@@ -31,7 +31,7 @@
     "type": "profession",
     "id": "afs_rating",
     "name": "Rating",
-    "description": "You were a junior crewmember within a freight spaceship.  Without any exceptionably valuable abilities, you spent your workshifts performing miscellaneous maintenance tasks within the ship.  As such, you are moderately skilled at performing all sorts of minor electrical and mechanical repairs, a skillset you hope remains useful planetside…",
+    "description": "You were a junior crewmember within a freight spaceship.  Without any exceptionally valuable abilities, you spent your workshifts performing miscellaneous maintenance tasks within the ship.  As such, you are moderately skilled at performing all sorts of minor electrical and mechanical repairs, a skillset you hope remains useful planetside…",
```
Comment (actual-nh) [inline data/mods/Aftershock/player/professions.json:34]:
```suggestion
    "description": "You were a junior crewmember on a freight spaceship.  Without any exceptionally valuable abilities, you spent your workshifts performing miscellaneous maintenance tasks within the ship.  As such, you are moderately skilled at performing all sorts of minor electrical and mechanical repairs, a skillset you hope remains useful planetside…",
```

File: data/mods/Aftershock/player/professions.json [hunk 6]
```diff
@@ -377,7 +377,7 @@
     "type": "profession",
     "id": "afs_cecaelia_scientist",
     "name": "Engineered Deep Sea Explorer",
-    "description": "Created with the exploration of the deep sea in mind Cecaelia are hybridized from cephalopod and fish dna.  While you were mostly spared from the corporate wars by your scientific proclivities, the polite niceties of the surface are mostly forgotten underwater.  You were on dry land when the Cataclysm occurred and you have no idea if it's safer here or beneath the seas.",
+    "description": "Created with the exploration of the deep sea in mind Cecaelia are hybridized from cephalopod and fish DNA.  While you were mostly spared from the corporate wars by your scientific proclivities, the polite niceties of the surface are mostly forgotten underwater.  You were on dry land when the Cataclysm occurred and you have no idea if it's safer here or beneath the seas.",
```
Comment (actual-nh) [inline data/mods/Aftershock/player/professions.json:380]:
```suggestion
    "description": "Created with the exploration of the deep sea in mind, Cecaelia are hybridized from cephalopod and fish DNA.  While you were mostly spared from the corporate wars by your scientific proclivities, the polite niceties of the surface are mostly forgotten underwater.  You were on dry land when the Cataclysm occurred and you have no idea if it's safer here or beneath the seas.",
```

**Review by actual-nh — COMMENTED** [review]
Good spotting; a few further edits.

**Comment by actual-nh** [comment]
> Thanks for making a great mod!

Agreed!

**Comment by casswedson** [comment]
_A few._ This is kinda hard >.>

**Comment by actual-nh** [comment]
> _A few._ This is kinda hard >.>

There is somewhat of a correlation between having one thing wrong and having other things wrong - a matter of how many people have checked over it, really. You spotted that there was at least one problem with each of these; I went in and found the other problems. (And I would never have gotten around to it but for your PR!)

**Comment by casswedson** [comment]
There is probably a lot more of this around, too bad hunting typos is not a popular activity, regardless thanks for the review.
I'll go hunt some more of these :)

**Pull request merged**